trillian/migrillian/core/controller.go

```diff
@@ -114,6 +160,8 @@ func (c *Controller) RunWhenMaster(ctx context.Context) error {
 // log. Returns if an error occurs, the context is canceled, or all the entries
 // have been transferred (in non-Continuous mode).
 func (c *Controller) Run(ctx context.Context) error {
+	metrics.isMaster.Set(1, c.label)
+	defer metrics.isMaster.Set(0, c.label)
```

Comment:
I think these would belong better in the caller, so that everything to do with mastership is visible in one place.

Reply:
Done.